srcpkgs/f3d/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'f3d'
+pkgname=f3d
+version=2.5.0
+revision=1
+build_style=cmake
+makedepends="vtk-devel"
+depends="vtk"
```

Comment:
This isn't necessary, the dependency will be automatically detected.

Reply:
![image](https://github.com/user-attachments/assets/bb58f377-268a-406f-8843-a55e9d615caa)

Reply:
Sorry, I should've clarified: I meant the `depends`, not the `makedepends`.